fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1346,6 +1348,9 @@ public void setEnableParallelMerge(boolean enableParallelMerge) {
     @VarAttr(name = ENABLE_MATERIALIZED_VIEW_PLAN_CACHE, flag = VariableMgr.INVISIBLE)
     private boolean enableMaterializedViewPlanCache = true;
 
+    @VarAttr(name = ENABLE_VIEW_BASED_MV_REWRITE)
+    private boolean enableViewBasedMvRewrite = true;
```

Comment:
Does it add significant cost to the rewrite procedure, or introduce risks ? 

If yes, how about setting the default value to false, until this feature got stable.

Reply:
change to default false